src/test/java/software/amazon/awssdk/crt/test/Mqtt5ClientTest.java

```diff
@@ -310,13 +227,12 @@ public void New_UC1() {
     @Test
     public void New_UC2() {
         skipIfNetworkUnavailable();
-        Assume.assumeTrue(mqtt5DirectMqttHost != null);
-        Assume.assumeTrue(mqtt5DirectMqttPort != null);
-        Assume.assumeTrue(mqtt5BasicAuthPassword != null);
-        Assume.assumeTrue(mqtt5BasicAuthUsername != null);
-        Assume.assumeTrue(mqtt5ProxyHost != null);
-        Assume.assumeTrue(mqtt5ProxyPort != null);
-
+        Assume.assumeTrue(AWS_TEST_MQTT5_DIRECT_MQTT_HOST != null);
```

Comment:
Trivial: I'm pretty sure you can use Assume.assumeNotNull() which takes I believe any number of arguments that would be great for these multiple checks that things aren't null.